spinalcordtoolbox/scripts/sct_deepseg.py

```diff
@@ -307,6 +307,23 @@ def get_parser(subparser_to_return=None):
             ({models.CROP_MESSAGE})
         """)
 
+        # Add possibility of having soft segmentation for the lesion_ms task
+        if task_name == 'lesion_ms':
+            params.add_argument(
+                "-soft-ms-lesion",
+                action="store_true",
+                help="If set, the model will output a soft segmentation (i.e. probability map) instead of a binary "
+                     "segmentation."
+            )
+
+        # Add possibility of segmenting on only 1 fold for quicker inference
+        if task_name == 'lesion_ms':
+            params.add_argument(
+                "-single-fold",
+                action="store_true",
+                help="If set, only 1 fold will be used for inference instead of the full 5-fold ensemble. This will speed up inference, but may reduce segmentation quality."
+            )
+
```

Comment:
I think we can move these arguments up so that they are next to the other "task-specific" args?